.github/workflows/cleanup-snapshots.yml

```diff
@@ -1,11 +1,6 @@
 name: Cleanup Snapshot Artifacts on a CRON Schedule
 
 on:  
-  schedule:
-    # Runs "At 00:00 on day-of-month 1 in every 3rd month." (see https://crontab.guru)
-#    - cron: '0 0 1 */3 *'
-    # Runs "At 03:00 every day" for test purposes only
-     - cron: '0 13 * * *'
```

Comment:
Don't we need this?

Reply:
I`m not sure if it is supposed to be scheduled in cpalce-fe or github-actions